data/production/categories.json

```diff
@@ -193,7 +193,8 @@
 			{
 				"name": "Icy conditions",
 				"id": 1011069,
```

Comment:
I think the ID Should be updated to "D" something?

Reply:
This is a weird one. Icy conditions is in citysourced, they just don’t want it to be submitted. So it has a legit citysourced id and doesn’t need a dummy. My thinking is only use dummies when there’s not an id, regardless of what we’re doing with disabling the form. What do you think?

Reply:
as long as we test this in dev, and it works as expected, I'm okay with this. Although it is a little confusing.

Reply:
Ok cool. Yeah it’s tested and works. I might change the approach when we add more items but for now this is what makes most sense to me.